project.Makefile

```diff
@@ -582,10 +582,8 @@ migrator:
 	$(RUN) create-migrator
 
 .PHONY: extract-study
-extract-study:
+extract-study: local/nmdc-sty-11-aygzgv51
```

Comment:
please rewrite the Python script so that both the study id and an output path (including filename) are required. Once we've done that, we can write linked makefile targets like this:

```make
local/nmdc-sty-11-aygzgv51.yaml:
	$(RUN) get-study-related-records \
		--api-base-url https://api-napa.microbiomedata.org \
		extract-study \
		--study-id nmdc:sty-11-aygzgv51 \
		--quick-test \
		--output-file $@

local/nmdc-sty-11-aygzgv51-validation.log: nmdc_schema/nmdc_materialized_patterns.yaml local/nmdc-sty-11-aygzgv51.yaml
	$(RUN) linkml-validate --schema $^ > $@
```

Reply:
we can add Makefile inference of the study id from the target name/output file path later

https://www.gnu.org/software/make/manual/html_node/Text-Functions.html

you can get a rough idea of that here:

- https://github.com/microbiomedata/nmdc-schema/blob/20010c597e1b5f00dcc194bf22cd7e4eb1710b29/project.Makefile#L542-L550

Reply:
We can consult with @cmungall if necessary. His Make skills are greater than mine, but ironically he often suggests something simpler.